src/main/java/com/snowflake/kafka/connector/Utils.java

```diff
@@ -102,6 +103,8 @@ public class Utils {
 
   private static final KCLogger LOGGER = new KCLogger(Utils.class.getName());
 
+  public static final String MDC_CONN_CTX_KEY = "connector.context";
```

Comment:
Lets add a comment here about this change and is available only after 2.3.0 
https://cwiki.apache.org/confluence/display/KAFKA/KIP-449%3A+Add+connector+contexts+to+Connect+worker+logs

Reply:
Lets add a comment here.